src/ontology/mondo-ingest.Makefile

```diff
@@ -369,3 +368,40 @@ slurp-%:
 	$(MAKE) slurp/$*.tsv
 
 slurp-all: slurp-omim slurp-doid slurp-ordo slurp-icd10cm slurp-icd10who slurp-ncit
+
+#############################
+###### Synchronization ######
+#############################
+sync/:
+	mkdir -p $@
+
+# TODO: @Nico
+tmp/mondo-ingest.db:
```

Comment:
@matentzn I believe you were going to complete these `make` goals:
- `tmp/mondo-ingest.db`
- `tmp/mondo-ingest-without-excluded.db`

Reply:
Defunct.
I've since refactored how this is being done. We're running the script per source, so we'll be passing the individual source's `.db`. I don't remember what that `without-excluded` thing is, but I doubt it is needed.